.travis.yml

```diff
@@ -21,12 +21,13 @@ addons:
 env:
     global:
         # Set defaults to avoid repeating in most cases
+        - ASTROPY_USE_SYSTEM_PYTEST=1
```

Comment:
@astrofrog , the point of this PR is **not** to use the bundled pytest, no? If I don't set this, current Travis CI will fall back on the bundled 2.8, so we're not really testing anything.